test/test_rest_vol.c

```diff
@@ -1493,8 +1493,6 @@ test_unused_file_API_calls(void)
         hid_t                obj_id;
         void                *file_handle;
 
-        if (H5Fis_accessible(filename, fapl_id) >= 0)
```

Comment:
Was there any particular reason for removing this call? Not that it matters since we want to deprecate the old tests anyway, but eventually we'll need to fully support `H5Fis_accessible` in the REST VOL if we don't already.

Reply:
If I understand right, this test was expecting `H5Fis_accessible` to always fail, which shouldn't be the case now that it's implemented.

Reply:
Ah yes, I probably could have figured that out from the test name if I'd looked.. disregard.